solr/CHANGES.txt

```diff
@@ -147,6 +149,8 @@ Bug Fixes
 * SOLR-16916: Use of the JSON Query DSL should ignore the defType parameter
   (Christina Chortaria, Max Kadel, Ryan Laddusaw, Jane Sandberg, David Smiley)
 
+* SOLR-16900: Add/fix wrong/missing "solr.home" to zkcli scripts  (Lamine Idjeraoui. Review by Eric Pugh & Jason Gerlowski)
```

Comment:
Unrelated edit?

Reply:
Yes not related to this PR.
But I forgot to add a change entry of another already merged PR, so I added it here to avoid a new PR just to add a change entry.
If you think it's not good idea never mind we can remove it.

Reply:
Yea, remove it from this PR, and do a separate commit (and PR) for the missing changes entry, with `SOLR-16900: ..` in the commit message - that will make it ping the JIRA issue to keep track of the commit history.

Reply:
done :)